test/unit/asrouter/ASRouterTargeting.test.js

```diff
@@ -80,29 +80,10 @@ describe("#CachedTargetingGetter", () => {
       context,
     });
 
-    assert.equal(stub.callCount, 6);
+    assert.equal(stub.callCount, 5);
     const calls = stub.getCalls().map(call => call.args[0]);
     const lastCall = calls[calls.length - 1];
-    assert.equal(lastCall.id, "FXA_1");
-  });
-  it("should return FxA message (is fallback)", async () => {
-    const messages = (await OnboardingMessageProvider.getUntranslatedMessages()).filter(
-      m => m.id !== "RETURN_TO_AMO_1"
-    );
```

Comment:
makes sense to remove, isn't this test same as above, wonder why we were filtering out RTAMO messages here?